chef-utils/spec/unit/dsl/platform_spec.rb

```diff
@@ -21,34 +21,67 @@
 
 def platform_reports_true_for(*args)
   args.each do |method|
-    it "reports true for #{method} on the module given a node" do
-      expect(described_class.send(method, node)).to be true
-    end
-    it "reports true for #{method} when mixed into a class with a node" do
-      expect(thing_with_a_node.send(method, node)).to be true
-    end
-    it "reports true for #{method} when mixed into a class with a run_context" do
-      expect(thing_with_a_run_context.send(method, node)).to be true
-    end
-    it "reports true for #{method} when mixed into a class with the dsl" do
-      expect(thing_with_the_dsl.send(method, node)).to be true
-    end
-    it "reports true for #{method} on the main class give a node" do
-      expect(ChefUtils.send(method, node)).to be true
+    if method.match?("centos_stream_platform")
```

Comment:
what am  I missing the if and else in this case are identical...

Reply:
The spec tests in this file by default goes through every single method and the `centos_stream_platform?` is the only one that doesn't need node data so I had to work around the tests so that they passed. If there's a better way to do this, please let me know.